desktop/extensions-sdk/dev/continuous-integration.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Continuous Integration"
```

Comment:
I know it's obvious in this context but just so we confirm to style and unpack the acronym, how about…

```suggestion
title: "Continuous Integration (CI)"
```